### PR TITLE
Update using-gpu-sanitizer.md

### PR DESCRIPTION
**Pull request by Rmalavally**
added the link text

### CODE REVIEW

**Review by samjwu — CHANGES_REQUESTED** [review]
please fix the Markdown linting error

Multiple consecutive blank lines

docs/conceptual/using-gpu-sanitizer.md:240 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2] https://github.com/DavidAnson/markdownlint/blob/v0.28.2/doc/md012.md

File: docs/conceptual/using-gpu-sanitizer.md [hunk 1]
```diff
@@ -235,9 +235,8 @@ $ rocgdb <path to application>
 
 ### Using ASan with a short HIP application
 
-Refer to the following example to use ASan with a short HIP application,
+Refer to the [example](https://github.com/Rmalavally/rocm-examples/blob/Rmalavally-patch-1/LLVM_ASAN/Using-Address-Sanitizer-with-a-Short-HIP-Application.md) to use ASan with a short HIP application.
```
Comment (b-sumner) [inline docs/conceptual/using-gpu-sanitizer.md:238]: The link here does not work when I paste it into a browser.

Comment (Rmalavally) [reply docs/conceptual/using-gpu-sanitizer.md:238]: Thanks, Brian. The original PR has been pending approval for a long time. Let me see how I can fix this for now.

**Review by Rmalavally — COMMENTED** [review]
I fixed the trailing spaces, added code blocks with bash. Is there anything else that i must know?

**Comment by Rmalavally** [comment]
> please fix the Markdown linting error
> 
> Multiple consecutive blank lines
> 
> docs/conceptual/using-gpu-sanitizer.md:240 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2] https://github.com/DavidAnson/markdownlint/blob/v0.28.2/doc/md012.md

We reformatted the entire file but still see errors. Let me check again.

**Comment by samjwu** [comment]
@Rmalavally I have fixed the Markdown linting errors here: https://github.com/ROCm/ROCm/pull/2970/commits/df3ad2d192330d877b4eb4417715499834f3f93a

The remaining issue is that "unallocated" is not in our spell checker's dictionary
This PR is to add it: https://github.com/ROCm/rocm-docs-core/pull/703

**Review by samjwu — APPROVED** [review]

**Pull request merged**